apps/randomcats/random_cats.star

```diff
@@ -0,0 +1,54 @@
+"""
+Applet: Random Cats
+Summary: Shows pictures of cats
+Description: Shows random pictures of cats/gifs of cats.
+Author: mrrobot245
+"""
+
+load("encoding/json.star", "json")
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+
+def main(config):
+    if (config.bool("gifs") == False):
+        imgSrc = http.get("https://cataas.com/cat?height=32").body()
```

Comment:
We'll have to add some caching here so the API call isn't made on every single run. In part to not overload the API, in part to simplify some of the caching that happens elsewhere in the Tidbyt backend. A ttl of say 20-30 seconds should do fine.

Reply:
Added!